Music

d-schn opened the pull request:
Added bg music as far as produced
bg music gets quieter as soon as Spinne music starts playing

d-schn committed:
Music changes

Added bg music, ~6min for now
Changed Spinne music to shorter, more energetic loop
bg music lowers as soon as Spinne music plays loud

d-schn committed:
Merge branch 'main' into music

d-schn committed:
Merge Main and balance music loudness

d-schn committed:
Merge branch 'main' into music

b-irsigler reviewed (approved):
Great, but could you improve the transition to the Spinne soundtrack?

d-schn committed:
Sound effects and music

- changed Spinne music to noise so it overlays better on bg music and changed attenuation
- added Gruene offer speech sfx
- changed Popup so screen time can be adjusted
- added death sfx
- adjusted sound levels

d-schn committed:
chopping sfx

sound effects for tree chopping and falling added

d-schn commented:
Changed Spinne music to noise and added SFX for Grüne offer, death, tree chopping and falling
Tested with @g-oelk 
Known issue: Spinne teleports closer to the player than max noise radius -> noise appears suddenly
